tables/tests/test_indexvalues.py

```diff
@@ -2296,6 +2296,11 @@ def test12a(self):
         self.assertFalse(t1var3.index.dirty)
         self.assertFalse(t1var4.index.dirty)
 
+        # TODO: IT IS DIRTY BECAUSE THIS FIXES THINGS FOR FINSV2aTestCase,
+        # which otherwise fails a test a few lines below!
+        for col in table1.colinstances.values():
+            col.reindex()
```

Comment:
This is marked as `TODO`.
I understand that it is a workaround to make the test pass but a fix would be needed somewhere in the PyTables code.
The workaround is for sure interesting because it provides good indications about the possible issue but, in principle, the problem is still there.
I would be not too keen to merge this without at least having a specific github issue for it.
Probably a better approach would be to mark this test as an expected failure for the time being, or something like that.

@ivilata , @FrancescAlted what is you position about this specific point?

Reply:
Yeah I have no idea what the right thing to do is, and this is probably not it! This was about as much as my limited knowledge would allow me to isolate the issue, there is probably a correct fix still remaining to be found / added so that this kludge can be removed.

Reply:
I agree with @avalentino about not merging without at least having an open issue and adding its number to the comment. However something definitely broke, though I don't know how serious it is. Probably @FrancescAlted has better insights since he knows the indexing code better.

Reply:
I am sorry, but I lately have personal duties that prevents me to put quality time on this. I'd appreciate if anybody else can go ahead and try to fix this one.  I think the NumPy team is helping other projects on-boarding with NumPy 2.0, so it is probably worth contacting them about this one.